src/core/server/saved_objects/export/types.ts

```diff
@@ -75,3 +78,18 @@ export interface SavedObjectsExportResultDetails {
     type: string;
   }>;
 }
+
+/**
+ * @public
+ */
+export interface SavedObjectsExportContext {
+  request: KibanaRequest;
+}
```

Comment:
As discussed in https://github.com/elastic/kibana/issues/84980, exposing only the request in the hook's context seems sufficient for our current needs.

Reply:
Should we move the `context` argument to be the first in the hook function signature? This would be more similar to our other APIs that have context arguments (eg. RouteHandler).